doc/library/tensor/nnet/conv.txt

```diff
@@ -190,12 +195,31 @@ TODO: Give examples on how to use these things! They are pretty complicated.
       please see the warning about a bug in CUDA 5.0 to 6.0
       in :func:`GpuCorrMM <theano.sandbox.cuda.blas.GpuCorrMM>`.
 
+    - :func:`Corr3dMM <theano.tensor.nnet.corr3d.Corr3dMM>`
+      This is a CPU-only 3d correlation implementation based on
+      the 2d version (:func:`CorrMM <theano.tensor.nnet.corr.CorrMM>`).
+      It does not flip the kernel. As it provides a gradient, you can use it as a
+      replacement for nnet.conv3d. For convolutions done on CPU,
+      nnet.conv3d will be replaced by Corr3dMM. To explicitly disable it, set
+      ``THEANO_FLAGS=optimizer_excluding=conv_gemm`` in your environment.
+
+    - :func:`dnn_conv3d <theano.sandbox.cuda.dnn.dnn_conv3d>` GPU-only
+      convolution using NVIDIA's cuDNN library. This requires that you have
+      cuDNN installed and available, which in turn requires CUDA 6.5 and a GPU
```

Comment:
I think we only support cuDNN v4 or more in the master, which requires CUDA 7.0 or later.